### DISCUSSION TITLE
Non HA Rook Ceph configuration

### DISCUSSION THREAD

**Discussion by vishakvinod** [rook/rook#7399] [Q&A] [answered]
Is there any documentation that would lead me to a minimal and non ha configuration for a local deployment ? I dont want to stress my local virtual machine with 20+ pods and moreover HA and failover is not a concern for local deployment for development and testing

**Comment by galexrt** [comment, marked as answer]
Have you tried the `*-test.yaml` files?
E.g., the `cluster-test.yml` will only start one mon, `etc. See https://github.com/rook/rook/tree/release-1.5/cluster/examples/kubernetes/ceph

**Reply by vishakvinod** [reply]
Thanks for the response. The `*-test.yaml` files looks promising to me. Jst one more thing to add, can the same be done using helm charts ?

**Reply by travisn** [reply]
Those single-node settings with *-test.yaml only apply to creating the CRs, which are not installed by the helm chart

**Reply by vishakvinod** [reply]
Okay. Thanks !
